app/src/util/types.ts

```diff
@@ -199,6 +199,7 @@ export interface UserInviteResponse {
 export enum Roles {
   User = "user",
   Admin = "admin",
+  OefAdmin = "oef_admin",
```

Comment:
Let's use the `Admin` role for global admin and create something else for organization/ project level admins.